keyboards/miniaxe/readme.md

```diff
@@ -0,0 +1,16 @@
+MiniAxe
+===
+
+![MiniAxe](https://i.imgur.com/1ApzrCz.jpg)
+
+Yet another split ortholinear keyboard with 3x5+3 keys.
+
+Keyboard Maintainer: [ka2hiro](https://github.com/ka2hiro) [@ka2hiro](https://twitter.com/ka2hiro) 
```

Comment:
This line needs another space at the end for the file to be rendered as intended.

```suggestion
Keyboard Maintainer: [ka2hiro](https://github.com/ka2hiro) [@ka2hiro](https://twitter.com/ka2hiro)  
```

Reply:
Added a space.